students/ghassan/lesson04/new_mailroom.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+import sys
+
+global donors
+donors = {
+        'Jessie': [100, 300, 500],
+        'James': [343, 22.11],
+        'Ford': [3000, 400]}
+
+
+def main_menu():
+    choice = input(
+        'Pick one:\n1) Send a thank you\n2) Create a Report\n\
+3) Send letters to everyone\n4) Quit\n')
+    return str(choice)
+
+
+def send_thankyou():
+    donor_name = input('Enter the donor\'s name: ')
```

Comment:
I would recommend using a `while` loop here, so that you can return to the main prompt after showing the list of donor names